TaskWithWritingInFile/TaskWithWritingInFileWay2/TaskWithWritingInFileWay2.cpp

```diff
@@ -0,0 +1,46 @@
+#include <windows.h>
+#include <string>
+#include <iostream>
+#include <errhandlingapi.h>
```

Comment:
The errhandlingapi.h must be a part of Windows.h (which you include in 1st line). You can remove and check the code can compile

Reply:
Removed